docs/rnn.rst

```diff
@@ -168,8 +168,12 @@ Iterate (or not)
 ----------------
 
 The ``apply`` method of a recurrent brick accepts an ``iterate`` argument,
-which defaults to ``True``. Setting it to ``False`` causes the ``apply`` method
-to compute only one step in the sequence.
+which defaults to ``True``. It is the reason for passing a 3-dimensional tensor
```

Comment:
This is true in this case: for recurrent models which transform states (input and output continuous features), the dimensionality is TxBxF. But recurrent decorator can be used for generative nets, like `SequenceGenerator`, in this case the outputs are integers (indexes) and have only 2 dimensions TxB.